database_wrapper.py

```diff
@@ -226,6 +226,60 @@ def __init__(self, config_file: str = 'config.json') -> None:
             msg = "config.json is missing {} field.".format('mysql')
             raise BadConfigFileError(msg)
 
+    '''Example:
+    >> > db = NimbusDatabase("config.json")
+    >> > db.get_property_from_related_entities(
+        ["firstName", "lastName", "ohRoom"],
+        "Professors", "OfficeHours", "professorId")
+    [("Foaad", "Khosmood", "14-213"), ("John", "Clements", "14-210"), ...]'''
+    def get_property_from_entity(
```

Comment:
Can we modify this function docstring to be under the first line of the function definition because it will help us generate docs better?

Example: https://google.github.io/styleguide/pyguide.html#doc-function-raises